tests/models/test_tpu.py

```diff
@@ -66,17 +67,18 @@ def test_model_tpu_cores_1(tmpdir):
     tpipes.run_model_test(trainer_options, model, on_gpu=False, with_hpc=False)
 
 
-@pytest.mark.parametrize("tpu_core", [1, 5])
+@pytest.mark.parametrize("devices", [1, 5])
```

Comment:
Let's keep this as tpu_core as well. We can address in a separate PR

Reply:
Ok, I trust your judgment

Reply:
I'll change it to `tpu_cores`

Reply:
Same as below :) thank you for addressing the comments

Reply:
Np! I've decided to use tpu_core just so we can find it again much more easily